test/tap/tests/unit/Makefile

```
@@ -8,6 +8,31 @@
 #
 # See: GitHub issue #5473 (Phase 2.1: Test Infrastructure Foundation)
 
+.DEFAULT_GOAL := all
+
+# The AWS SDK must live in the v4-only AWS IAM plugin, never in the core
+# daemon.  Kept as an explicit build-artifact gate because it exercises the
+# actual linker output rather than a source-code convention.
+.PHONY: aws_plugin_linkage-t
+aws_plugin_linkage-t:
```

checkmake (0.3.2)

[warning] Target body for "aws_plugin_linkage-t" exceeds allowed length of 5 lines (7).

(maxbodylength)